ucte/ucte-converter/src/main/java/com/powsybl/ucte/converter/UcteExporter.java

```diff
@@ -0,0 +1,1072 @@
+     * @param ucteNetwork The target UcteNetwork
+     */
+    private void convertTwoWindingsTransformers(Network network, UcteNetwork ucteNetwork) {
+        LOGGER.debug("Converting two winding transformers");
+        network.getTwoWindingsTransformerStream()
+                .forEach(twoWindingsTransformer -> convertTwoWindingsTransformer(ucteNetwork, twoWindingsTransformer));
+        LOGGER.debug("two winding transformers converted");
+    }
+
+    /**
+     * Converts the {@link TwoWindingsTransformer} into a {@link UcteTransformer} and adds it to the ucteNetwork.
+     * Also creates the adds the linked {@link UcteRegulation}
+     *
+     * @param ucteNetwork The target UcteNetwork
+     * @param twoWindingsTransformer The two windings transformer we want to convert
+     * @see UcteExporter#convertRegulation(UcteNetwork, UcteElementId, TwoWindingsTransformer)
+     */
+    private void convertTwoWindingsTransformer(UcteNetwork ucteNetwork, TwoWindingsTransformer twoWindingsTransformer) {
+        Terminal terminal1 = twoWindingsTransformer.getTerminal1();
+        Terminal terminal2 = twoWindingsTransformer.getTerminal2();
+
+        Optional<Country> country1 = terminal1.getVoltageLevel().getSubstation().getCountry();
+        Optional<Country> country2 = terminal2.getVoltageLevel().getSubstation().getCountry();
+
+        if (!country1.isPresent() || !country2.isPresent()) {
+            throw new UcteException("Two windings transformer " + twoWindingsTransformer.getId() + NO_COUNTRY_FOUND);
+        }
+
+        UcteNodeCode ucteNodeCode1 = createUcteNodeCode(
+                terminal1.getBusBreakerView().getConnectableBus().getId(),
+                terminal1.getVoltageLevel(),
+                country1.get().toString()
+        );
+
+        UcteNodeCode ucteNodeCode2 = createUcteNodeCode(
+                terminal2.getBusBreakerView().getConnectableBus().getId(),
+                terminal2.getVoltageLevel(),
+                country2.get().toString()
+        );
+
+        UcteElementId ucteElementId = convertUcteElementId(ucteNodeCode2, ucteNodeCode1, twoWindingsTransformer.getId(), terminal1, terminal2);
+        String elementName = twoWindingsTransformer.getProperties().getProperty(ELEMENT_NAME_PROPERTY_KEY, null);
+        UcteTransformer ucteTransformer = new UcteTransformer(
+                ucteElementId,
+                UcteElementStatus.fromCode(1),
+                (float) twoWindingsTransformer.getR(),
+                (float) twoWindingsTransformer.getX(),
+                (float) twoWindingsTransformer.getB(),
+                (int) twoWindingsTransformer.getCurrentLimits2().getPermanentLimit(),
+                elementName,
+                (float) twoWindingsTransformer.getRatedU2(),
+                (float) twoWindingsTransformer.getRatedU1(),
+                100,        //TODO Find a representation for the nominal power
```

Comment:
Is this default value OK?

Reply:
It was chosen kind of randomly, It has to be changed anyway with a real value.

Reply:
Does this mean that if another value was used in the initial Ucte file, it will be changed to this default value ?

Reply:
Since it's not stored when we import them, yes.

Reply:
Do we put this in a property to avoid losing it?